File: eShelvesAPI/eShelvesAPI/ViewModels/KnjigaDetaljiVM.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace eShelvesAPI.ViewModels
{
    public class KnjigaDetaljiVM
    {
        public string Naslov { get; set; }
        public int AutorID { get; set; }
        public string NazivAutora { get; set; }
        public double ProsjecnaOcjena { get; set; }
        public string ISBN { get; set; }
        public string Opis { get; set; }

        public List<OcjenaInfoVM> OcjenaInfoVMs { get; set; }

        public class OcjenaInfoVM
        {

            public int KorisnikId { get; set; }
            public string username { get; set; }
            public int Ocjena { get; set; }
            public string Opis { get; set; }
        }
    }
}

File: eShelvesAndroid/app/src/main/java/com/readmore/tonka/models/KnjigaDetaljiVM.java
package com.readmore.tonka.models;

import java.io.Serializable;
import java.util.List;

/**
 * Created by anton on 20.05.2015..
 */
public class KnjigaDetaljiVM {
    public String Naslov;
    public int AutorID;
    public String NazivAutora;
    public float ProsjecnaOcjena;
    public String ISBN;
    public String Opis;

    public List<OcjenaInfoVM> OcjenaInfoVMs;

    public class OcjenaInfoVM implements Serializable{
        public int KorisnikId;
        public String username;
        public int Ocjena;
        public String Opis;
    }
}
